package.json

```diff
@@ -1,57 +1,60 @@
 {
   "name": "react-hook-videojs",
-  "version": "2.1.0-beta.1",
+  "version": "2.1.0-beta.3",
```

Comment:
please remove version update from here, we'll deal with version upgrade separately